.gitignore

```diff
@@ -1,2 +1,4 @@
 *.class
 Thumbs.db
+/.metadata/
+/bin/
```

Comment:
@ThatIsAUsername do we even have a policy on changes to .gitignore?
I just hid it so I don't have to mess with it. thoughts?

Reply:
It's pretty reasonable to ignore /bin/. I'm semi surprised we didn't have that on the list already.